exchanges/graphcache/src/default-storage/index.ts

```diff
@@ -54,12 +54,10 @@ export const makeDefaultStorage = (opts?: StorageOptions): DefaultStorage => {
   const serializeBatch = (): string => {
     let data = '';
     for (const key in batch) {
-      const value = batch[key];
-      data += `${stringifyVariables(key)}:${
-        value !== undefined ? stringifyVariables(value) : 'null'
-      },`;
+      if (batch[key] !== undefined) {
```

Comment:
This isn’t quite right I believe, because we do need to ensure that this deletes / overrides old values.
So we may want to specially encode `undefined`